src/java.desktop/share/classes/javax/swing/plaf/basic/BasicLookAndFeel.java

```diff
@@ -25,60 +25,31 @@
 
 package javax.swing.plaf.basic;
 
-import java.awt.Font;
-import java.awt.Color;
-import java.awt.SystemColor;
-import java.awt.event.*;
-import java.awt.Insets;
-import java.awt.Component;
-import java.awt.Container;
-import java.awt.FocusTraversalPolicy;
-import java.awt.AWTEvent;
-import java.awt.Toolkit;
-import java.awt.Point;
-import java.net.URL;
-import java.io.*;
-import java.awt.Dimension;
-import java.awt.KeyboardFocusManager;
-import java.security.AccessController;
-import java.security.PrivilegedAction;
-import java.util.*;
-import java.lang.reflect.*;
-import javax.sound.sampled.*;
-
 import sun.awt.AppContext;
 import sun.awt.SunToolkit;
 import sun.swing.SwingAccessor;
 import sun.swing.SwingUtilities2;
 import sun.swing.icon.SortArrowIcon;
 
-import javax.swing.LookAndFeel;
-import javax.swing.AbstractAction;
-import javax.swing.Action;
-import javax.swing.ActionMap;
-import javax.swing.BorderFactory;
-import javax.swing.JComponent;
-import javax.swing.ImageIcon;
-import javax.swing.UIDefaults;
-import javax.swing.UIManager;
-import javax.swing.KeyStroke;
-import javax.swing.JTextField;
-import javax.swing.DefaultListCellRenderer;
-import javax.swing.FocusManager;
-import javax.swing.LayoutFocusTraversalPolicy;
-import javax.swing.SwingUtilities;
-import javax.swing.MenuSelectionManager;
-import javax.swing.MenuElement;
-import javax.swing.border.*;
+import javax.sound.sampled.*;
+import javax.swing.*;
```

Comment:
Fix looks ok.
But , please replace the wildcard imports with specific ones

Reply:
Thank you!